src/components/Feed/Feed.css

```diff
@@ -1,17 +1,17 @@
 main {
-    animation: soft-rainbow 20s infinite; 
+    animation: soft-rainbow 20s infinite;
 }
 
 @keyframes soft-rainbow {
```

Comment:
I see you moved the keyframe! Nice, I think going forward we should make the keyframe match the wave colors